projects/Apps/common/src/logging.ts

```diff
@@ -35,6 +35,8 @@ export interface MallardLogFormat {
     version: string
     buildNumber: string
     release_channel: ReleaseChannel
+    selectedEdition: string
+    defaultEdition: string | null
```

Comment:
A default edition can only be `RegionalEdition` and the selected edition can be `RegionalEdition | SpecialEdition`

Reply:
There is also no null case based on `defaultEditionDecider`

Reply:
For the purpose of logging I was thinking we would just want the edition eg. "daily-edition", so maybe `Edition` would be best for both, unless we want the other metadata? 
To avoid the null case should I be using something other than `getDefaultEdition` which accesses the cache?

Reply:
As discussed on chat with @jimhunty I've used `Edition` and left the null specifier there to make TS happy.